pyramid_zipkin/tween.py

```diff
@@ -8,17 +8,43 @@
 from pyramid_zipkin.request_helper import create_zipkin_attr
 from pyramid_zipkin.request_helper import get_binary_annotations
 
-
-ZipkinSettings = namedtuple('ZipkinSettings', [
+ZipkinTuple = namedtuple('ZipkinSettings', [
     'zipkin_attrs',
     'transport_handler',
     'service_name',
     'span_name',
     'add_logging_annotation',
     'report_root_timestamp',
+    'host',
+    'port'
 ])
 
 
+def make_zipkin_settings(
+        zipkin_attrs=None,
+        transport_handler=None,
+        service_name=None,
+        span_name=None,
+        add_logging_annotation=None,
+        report_root_timestamp=None,
+        host=None,
+        port=None
+):
+    return ZipkinTuple(
+        zipkin_attrs,
+        transport_handler,
+        service_name,
+        span_name,
+        add_logging_annotation,
+        report_root_timestamp,
+        host,
+        port
+    )
+
+
+ZipkinSettings = make_zipkin_settings
```

Comment:
You're right. I was just afraid of breaking the api but I looked into Yelp Pyramid and Yelp Main and they don't access ZipkinSettings so I think we're good?